index.php

```diff
@@ -18,37 +18,42 @@
     }
 
     $tableHeader = '<thead>
-                        <th>NAME</th>
+                        <th>Player</th>
                         <th><A href="?sortby=0&sortorder=desc" style="color: white">Track Length</A></th>
-                        <th><A href="?sortby=1&sortorder=desc" style="color: white">#Y</A> / <A href="?sortby=6&sortorder=desc" style="color: white">#T</A></th>
+                        <th><A href="?sortby=1&sortorder=desc" style="color: white">Switches</A></th>
+                        <th><A href="?sortby=6&sortorder=desc" style="color: white">Trees Removed</A></th>
                         <th><A href="?sortby=2&sortorder=desc" style="color: white">Locos</A></th>
                         <th><A href="?sortby=3&sortorder=desc" style="color: white">Carts</A></th>
                         <th><A href="?sortby=4&sortorder=desc" style="color: white">Slope</A></th>
+                        <th><A href="?sortby=4&sortorder=desc" style="color: white">Shared (2 Days)</th>
                     </thead>';
 
-    function mysort($a, $b){
-        global $db;
-        $x=1;
-        if(strtolower($db[substr($a, 5, -5) . '.sav'][$_GET['sortby']]) == strtolower($db[substr($b, 5, -5) . '.sav'][$_GET['sortby']])){
-            return 0;
-        }
-        if(strtolower($db[substr($a, 5, -5) . '.sav'][$_GET['sortby']]) > strtolower($db[substr($b, 5, -5) . '.sav'][$_GET['sortby']])){
-            $x=-1;
-        } else {
-            $x=1;
-        }
-        if($_GET['sortorder']=='desc'){
-            return $x;
-        } else {
-            return -$x;
-        }
-    }
+// Removed sort function for different variation
+//
+//    function mysort($a, $b){
+//        global $db;
+//        $x=1;
+//        if(strtolower($db[substr($a, 5, -5) . '.sav'][$_GET['sortby']]) == strtolower($db[substr($b, 5, -5) . '.sav'][$_GET['sortby']])){
+//            return 0;
+//        }
+//        if(strtolower($db[substr($a, 5, -5) . '.sav'][$_GET['sortby']]) > strtolower($db[substr($b, 5, -5) . '.sav'][$_GET['sortby']])){
+//            $x=-1;
+//        } else {
+//            $x=1;
+//        }
+//        if($_GET['sortorder']=='desc'){
+//            return $x;
+//        } else {
+//            return -$x;
+//        }
+//    }
+//
 
```

Comment:
This function is needed that the links on the tableheaders work.
why got it removed?